modules/kargoBidAdapter.js

```diff
@@ -284,6 +336,83 @@ export const spec = {
 
       triggerPixel(timeoutRequestUrl);
     } catch (e) {}
+  },
+
+  getImpression(bid) {
+    const imp = {
+      id: bid.bidId,
+      tid: bid.transactionId,
+      pid: bid.params.placementId,
+      code: bid.adUnitCode
+    }
+
+    if (bid.floor > 0) {
```

Comment:
@jsadwith Perhaps this is something I have overlooked but the accompany examples with this Jira do not show how floor is to be populated. Any advice/pointers?

Reply:
@njflynn good question and good catch. Previously, the adapter was just hardcoding the floor to `0` and Kraken pulled it from prebidRequest like this...

```
if prebidRequest.FloorData.BidFloor != 0 {
			bidRequest.BidFloor = prebidRequest.FloorData.BidFloor
		}
```

BidFloor is pulled from the JSON param `floorMin`. So let's use `bid.floorData.floorMin`. I'll update JIRA and the example to reflect this.